src/components/overview/Description.jsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import styles from './overview.module.css'
 
 
-const Description = (props) => {
+const Description = ({productInfo}) => {
```

Comment:
for better practice, try using context if available instead of passing the element down